### PR TITLE
delete yarn.lock and modify function userRepo

### PR DESCRIPTION
**Pull request by maximangetman**

### CODE REVIEW

File: lib/user-repo.js [hunk 1]
```diff
@@ -4,22 +4,24 @@ const { GH_GENERAL } = require('@gh-conf/gh-conf-constants');
 
 const ParseConfig = require('./parse-config');
 
+const getCurrRepo = () => {
```
Comment (arshadkazmi42) [inline lib/user-repo.js]: Can you revert these changes?
We do not want to merge two PR in one.
Just have the removing yarn.lock file in this PR

Comment (maximangetman) [reply lib/user-repo.js]: ok, sorry
